lib/asciidoctor/extensions.rb

```diff
@@ -1,3 +1,5 @@
+require 'asciidoctor'.to_s unless defined? Asciidoctor
```

Comment:
@mojavelinux just for my curiosity, why are you using the `to_s` on a String ? As far as I know, `to_s` returns the String representation of an Object, so `'asciidoctor'` and `'asciidoctor'.to_s` should be equivalent no ?